Toolbar/Toolbar.jsx

```diff
@@ -79,7 +79,7 @@ class ToolbarIcon extends MaterialComponent {
     super();
     this.componentName = "toolbar__icon";
     if (props.menu) {
-      this.componentName += "--menu";
+      this.componentName += "-menu";
```

Comment:
The final output should be ```mdc-toolbar__menu-icon```, not ```mdc-toolbar_icon-menu```.

https://github.com/material-components/material-components-web/blob/master/CHANGELOG.md#breaking-changes

Reply:
Maybe a new component would be cleaner than completely changing ```componentName```?